packages/l/libmikmod/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : libmikmod
 version    : 3.3.11.1
-release    : 11
+release    : 12
 source     :
-    - https://netix.dl.sourceforge.net/project/mikmod/libmikmod/3.3.11.1/libmikmod-3.3.11.1.tar.gz : ad9d64dfc8f83684876419ea7cd4ff4a41d8bcd8c23ef37ecb3a200a16b46d19
+    - https://sourceforge.net/projects/mikmod/files/libmikmod/3.3.11.1/libmikmod-3.3.11.1.tar.gz : ad9d64dfc8f83684876419ea7cd4ff4a41d8bcd8c23ef37ecb3a200a16b46d19
+homepage   : https://mikmod.sourceforge.net
 license    :
     - LGPL-2.0
```

Comment:
Please fix these licenses to correct format.

Reply:
Hmm, their website links to LGPL-3, should I change those two licenses to that?

Reply:
Ah nevermind, the tarball source has 2 and 2.1. Will fix!